lib/Constants.js

```diff
@@ -370,5 +370,241 @@ module.exports = {
+      specificTypes: {
+        0x01: 'specific_type_routing_sensor_binary'
+      }
+    },
+    0x21: {
+      genericType: 'generic_type_sensor_multilevel',
+      specificTypes: {
+        0x01: 'specific_type_routing_sensor_multilevel',
+        0x02: 'specific_type_chimney_fan'
+      }
+    },
+    0x02: {
+      genericType: 'generic_type_static_controller',
+      specificTypes: {
+        0x01: 'specific_type_pc_controller',
+        0x02: 'specific_type_scene_controller',
+        0x03: 'specific_type_static_installer_tool',
+        0x04: 'specific_type_set_top_box',
+        0x05: 'specific_type_sub_system_controller',
+        0x06: 'specific_type_tv',
+        0x07: 'specific_type_gateway'
+      }
+    },
+    0x10: {
+      genericType: 'generic_type_switch_binary',
+      specificTypes: {
+        0x01: 'specific_type_power_switch_binary',
+        0x03: 'specific_type_scene_switch_binary',
+        0x04: 'specific_type_power_strip',
+        0x05: 'specific_type_siren',
+        0x06: 'specific_type_valve_open_close',
+        0x02: 'specific_type_color_tunable_binary',
+        0x07: 'specific_type_irrigation_controller'
+      }
+    },
+    0x11: {
+      genericType: 'generic_type_switch_multilevel',
+      specificTypes: {
+        0x05: 'specific_type_class_a_motor_control',
+        0x06: 'specific_type_class_b_motor_control',
+        0x07: 'specific_type_class_c_motor_control',
+        0x03: 'specific_type_motor_multiposition',
+        0x01: 'specific_type_power_switch_multilevel',
+        0x04: 'specific_type_scene_switch_multilevel',
+        0x08: 'specific_type_fan_switch',
+        0x02: 'specific_type_color_tunable_multilevel'
+      }
+    },
+    0x12: {
+      genericType: 'generic_type_switch_remote',
+      specificTypes: {
+        0x01: 'specific_type_remote_binary',
+        0x02: 'specific_type_remote_multilevel',
+        0x03: 'specific_type_remove_toggle_binary',
+        0x04: 'specific_type_remote_toggle_multilevel'
+      }
+    },
+    0x13: {
+      genericType: 'generic_type_switch_toggle',
+      specificTypes: {
+        0x01: 'specific_type_switch_toggle_binary',
+        0x02: 'specific_type_switch_toggle_multilevel'
+      }
+    },
+    0x08: {
+      genericType: 'generic_type_thermostat',
+      specificTypes: {
+        0x03: 'specific_type_setback_schedule_thermostat',
+        0x05: 'specific_type_setback_thermostat',
+        0x04: 'specific_type_setpoint_thermostat',
+        0x02: 'specific_type_thermostat_general',
+        0x06: 'specific_type_thermostat_general_v2',
+        0x01: 'specific_type_thermostat_heating',
+      }
+    },
+    0x16: {
+      genericType: 'generic_type_ventilation',
+      specificTypes: {
+        0x01: 'specific_type_residential_hrv'
+      }
+    },
+    0x09: {
+      genericType: 'generic_type_windows_covering',
+      specificTypes: {
+        0x01: 'specific_type_simple_window_covering'
+      }
+    },
+    0x15: {
```

Comment:
Dup

Reply:
Fixed in dd59a255189252f55124f938b40d0eb2cc52cfb1